app/graphql/types/query_type.rb

```diff
@@ -10,7 +10,7 @@ class QueryType < Types::BaseObject
     end
 
     def user_id_query(kwargs, searcher=Searchers::Complete)
-      if context[:current_user].to_i == kwargs[:user_id].to_i
+      if context[:admin] || context[:current_user].to_i == kwargs[:user_id].to_i
```

Comment:
what happens if i manage to craft a a string like `'not-a-number'` as the user_id here?

Reply:
So should I compare them as strings? The graphql query passes the user_id down as a string so I need to convert either way.